pkg/process/checks/process_events_consumer.go

```diff
@@ -0,0 +1,163 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
What do you think of moving this file to `/pkg/process/events` to keep everything event related into the same location?

Reply:
Good idea, will do!